src/main/java/com/ibm/g11n/pipeline/client/NewTranslationRequestData.java

```diff
@@ -0,0 +1,258 @@
+     * status is changed, an e-mail notification will be sent to the addresses.
+     * 
+     * @return  A list of the requester's e-mail addresses.
+     */
+    public List<String> getEmails() {
+        return emails;
+    }
+
+    /**
+     * Sets a list of the requester's contact phones. Optional.
+     * 
+     * @param phones    A list of the requester's contact phones.
+     * @return  This object.
+     */
+    public NewTranslationRequestData setPhones(List<String> phones) {
+        this.phones = phones;
+        return this;
+    }
+
+    /**
+     * Returns a list of the requester's contact phones.
+     * 
+     * @return  A list of the requester's contact phones.
+     */
+    public List<String> getPhones() {
+        return phones;
+    }
+
+    /**
+     * Sets a set of the industry domains representing the contents.
+     * @param domains   A set of the industry domains representing the contents.
+     * @return  This object.
+     */
+    public NewTranslationRequestData setDomains(EnumSet<IndustryDomain> domains) {
+        this.domains = domains;
+        return this;
+    }
+
+    /**
+     * Returns a set of the industry domains representing the contents. Optional.
+     * 
+     * @return  A set of industry domains representing the contents.
+     */
+    public EnumSet<IndustryDomain> getDomains() {
+        return domains;
+    }
+
+    /**
+     * Sets the translation notes. Optional.
+     * 
+     * @param notes The translation notes.
+     * @return  This object.
+     */
+    public NewTranslationRequestData setNotes(List<String> notes) {
+        this.notes = notes;
+        return this;
+    }
+
+    /**
+     * Returns the translation notes.
+     * 
+     * @return  The translation notes.
+     */
+    public List<String> getNotes() {
+        return notes;
+    }
+
+    /**
+     * Sets a map containing the metadata of this translation request represented
+     * by key-value pairs. Optional.
+     * 
+     * @param metadata  A map containing the metadata of this translation request represented
+     * by key-value pairs.
+     * @return  This object.
+     */
+    public NewTranslationRequestData setMetadata(Map<String, String> metadata) {
+        this.metadata = metadata;
+        return this;
+    }
+
+    /**
+     * Returns a map containing the metadata of this translation request represented
+     * by key-value pairs.
+     * 
+     * @return A map containing the metadata of this translation request represented
+     * by key-value pairs.
+     */
+    public Map<String, String> getMetadata() {
+        return metadata;
+    }
+
+    /**
+     * Sets whether this new translation request will be submitted immediately
+     * after creation. The default value is false.
+     * 
+     * @param submit    Sets whether this new translation request will be submitted
+     *                  immediately.
+     * @return  This object.
+     */
+    public NewTranslationRequestData setSubmit(boolean submit) {
```

Comment:
This API is aligned to the REST API's behavior. When a new TR is created without status field, the REST server automatically set status = DRAFT. So, default value is DRAFT.

I don't want make the user API to set any status. Technically, user can only change the status from DRAFT to SUBMITTED (and creating a new TR as born "SUBMITTED").